bundler/lib/bundler/templates/newgem/.github/workflows/main.yml.tt

```diff
@@ -0,0 +1,18 @@
+name: Ruby
+
+on: [push,pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby <%= ::Gem::Version.new(RUBY_VERSION).segments[0..1].join(".") %>
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: <%= ::Gem::Version.new(RUBY_VERSION).segments[0..1].join(".") %>.x
+    - name: Build and test with Rake
+      run: |
+        gem install bundler -v <%= Bundler::VERSION %>
+        bundle install --jobs 4 --retry 3
```

Comment:
I would remove the `--jobs 4`, since the next version of `bundler` will autochoose jobs by default. We can also remove `--retry 3` since it's the default.